web-ui/src/components/reviews/periods/ReviewPeriodStepper.jsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import Stepper from '@mui/material/Stepper';
+import Step from '@mui/material/Step';
+import StepLabel from '@mui/material/StepLabel';
+
+const steps = [
+  'Review Period Is Open',
+  'Review Period Is Closed'
+];
+
+export default function HorizontalLinearAlternativeLabelStepper({reviewPeriod}) {
```

Comment:
Here we pass in the full reviewPeriod object. If we only need to pass in the status 'open', this could be simplified.